L1Trigger/L1CaloTrigger/python/test_L1EGCrystalClusterEmulator_cfg.py

```diff
@@ -0,0 +1,76 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.StandardSequences.Eras import eras
+
+process = cms.Process("L1AlgoTest",eras.Phase2_trigger)
+
+process.load('Configuration.StandardSequences.Services_cff')
+process.load("FWCore.MessageService.MessageLogger_cfi")
+process.load('Configuration.EventContent.EventContent_cff')
+process.MessageLogger.categories = cms.untracked.vstring('L1EGRateStudies', 'FwkReport')
+process.MessageLogger.cerr.FwkReport = cms.untracked.PSet(
+   reportEvery = cms.untracked.int32(1)
+)
+
+process.maxEvents = cms.untracked.PSet( input = cms.untracked.int32(10) )
+
+process.source = cms.Source("PoolSource",
+   # Set to do test run on official Phase-2 L1T Ntuples
+   fileNames = cms.untracked.vstring('file:root://cmsxrootd.fnal.gov///store/mc/PhaseIISpring17D/SingleE_FlatPt-8to100/GEN-SIM-DIGI-RAW/PU200_90X_upgrade2023_realistic_v9-v1/120000/04B4BF1D-1E2C-E711-BE1C-7845C4FC39D1.root'),
+   inputCommands = cms.untracked.vstring(
+                    "keep *",
+                    "drop l1tEMTFHitExtras_simEmtfDigis_CSC_HLT",
+                    "drop l1tEMTFHitExtras_simEmtfDigis_RPC_HLT",
+                    "drop l1tEMTFTrackExtras_simEmtfDigis__HLT",
+   )
+)
+
+# All this stuff just runs the various EG algorithms that we are studying
+                         
+# ---- Global Tag :
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag, '100X_upgrade2023_realistic_v1', '')
+
+# Choose a 2023 geometry!
+process.load('Configuration.Geometry.GeometryExtended2023D17Reco_cff')
```

Comment:
has this been tested with the D49 geometry used for the HLT TDR?

Reply:
This file is an expert test configuration 

And yes, this code has been tested in the phase2 l1t integration branch with the recipe for D49 and D41 and has been used by the HLT people already to run on the HLT Phase2 samples.